front/components/assistant/AssistantList.tsx

```diff
@@ -0,0 +1,56 @@
+import { AssistantPreview } from "@dust-tt/sparkle";
+import type { LightAgentConfigurationType } from "@dust-tt/types";
+import { useRouter } from "next/router";
+import React from "react";
+
+interface AssistantListProps {
+  agents: LightAgentConfigurationType[];
+  handleAssistantClick: (agent: LightAgentConfigurationType) => void;
+}
+
+export function AssistantList({
```

Comment:
Any reason to not have all the tabs and search logic here? It would declutter the new page a lot!

Reply:
:factory_worker: :mountain:

Reply:
Ok, did it